components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import AddParticipants from "./AddParticipants";
+
+function Form() {
+  const [title, setTitle] = useState("");
+  const [amount, setAmount] = useState("");
+
+  return (
+    <>
+      <Title>Create New Bill</Title>
+      <Container>
```

Comment:
Your form currently has no submit handler. Your feature is not complete yet. You should change this before moving further.

Reply:
Using the preview and trying to submit something is somewhat confusing, because of the browser message I get. But I see that this is how you intended it.